crates/egui_extras/src/table.rs

```diff
@@ -768,6 +798,9 @@ pub struct TableBody<'a> {
     /// If we find the correct row to scroll to,
     /// this is set to the y-range of the row.
     scroll_to_y_range: &'a mut Option<Rangef>,
+
+    hovered_row: Option<usize>,
+    hovered_row_id: egui::Id,
```

Comment:
this is a bit confusing name - it sounds like the id of the row that is being hover, rather than the id of the storage of the hovered row index.